src/js/videojs.wavesurfer.js

```diff
@@ -574,6 +569,7 @@ class Wavesurfer extends Plugin {
      */
     onWaveFinish() {
         this.log('Finished playback');
+        this.setCurrentTime();
```

Comment:
unnecessary call.

Reply:
withouth this, it showed me a incorrect time, when the track has ended. even withouth the counter.